tests/functional/u/unused/unused_name_in_string_literal_type_annotation.py

```diff
@@ -21,6 +21,11 @@ def example3(_: "os.PathLike[str]") -> None:
 def example4(_: "PathLike[str]") -> None:
     """unused-import shouldn't be emitted for PathLike."""
 
+# pylint shouldn't crash with the following strings in a type annotation context
+example5: Set[""]
```

Comment:
This was added to increase the test coverage. Should pylint emit a message when it finds an invalid type annotation?

Reply:
Let's leave that to `mypy` and the like 😄